### ISSUE TITLE
Radio inputs doesnt fill with old input

### ISSUE THREAD

**Issue by Fr0ntman** [spatie/laravel-html#92]
Hello! When try:
```php
return back()->withInput(
            $request->all()
        );
```
radio inputs like:
```php
html()->radio('dest', true)->value('mtr');
html()->radio('dest', false)->value('dtr');
```
doesnt fill correctly.
This doesnt work:
```php
return $this->input('radio', $name, $value)
            ->attributeIf((bool) $this->old($name, $checked), 'checked');
```
This works:
```php
html()->radio('dest', true)->value('mtr')->checked( old('dest') === 'mtr' );
html()->radio('dest', false)->value('dtr')->checked( old('dest') === 'dtr' );
```

**Comment by sebastiandedeyne** [comment]
That's actually the expected behaviour! Reasoning about this is described in the docs: https://docs.spatie.be/laravel-html/v2/general-usage/core-concepts#the-difference-between-builder-params-and-element-methods

**Comment by Fr0ntman** [comment]
But this doesnt work too:
```php
html()->radio('dest', true, 'mtr');
html()->radio('dest', false, 'dtr');
```
What i do wrong? Or out of the box we havent filling radio inputs with old input?

**Comment by sebastiandedeyne** [comment]
Released `2.19.1` that should fix this issue, could you give it a try?
